### PR TITLE
Replaced smart_* by force_* calls whenever possible

### PR DESCRIPTION
**Pull request by claudep**

### CODE REVIEW

**Comment by charettes** [comment]
You might want to investigate all usages of `smart_(text|bytes)` as they are a couple ones that could be replaced by `force` one. The `django/core/serializers/xml_serializer.py` is a good example.

**Comment by claudep** [comment]
Yes, many `smart` could be replaced by `force`. Some are obvious, while for others, it's more tricky to know if some lazy object should be kept lazy. Doing it piecemeal is maybe not the better approach, agreed.

**Comment by claudep** [comment]
Closing it meanwhile, until I can find the time for a more thorough investigation.

**Comment by claudep** [comment]
Reopening as I'll push an update

File: django/contrib/staticfiles/management/commands/collectstatic.py [hunk 1]
```diff
@@ -232,9 +232,9 @@ def clear_dir(self, path):
             fpath = os.path.join(path, f)
             if self.dry_run:
                 self.log("Pretending to delete '%s'" %
-                         smart_text(fpath), level=1)
+                         force_text(fpath), level=1)
```
Comment (timgraham) [inline django/contrib/staticfiles/management/commands/collectstatic.py]: could move this to the previous line

**Comment by charettes** [comment]
Thanks for the investigation Claude!

**Pull request merged**